explore/src/main/scala/explore/observationtree/ObsSummaryTile.scala

```diff
@@ -191,305 +192,307 @@ object ObsSummaryTile:
+          r => r.coordsAtVizTime.map(_.dec)
+        )
+          .setCell(_.value.map(MathValidators.truncatedDec.reverseGet).orEmpty)
+          .sortable,
+        // TODO: TimingColumnId
+        // TODO: SEDColumnId
+        ColDef(
+          SEDColumnId,
+          v =>
+            v.value
+              .fold(_.targetWithId.target.some, _.targetWithId.map(_.target))
+              .flatMap(Target.sidereal.getOption)
+              .flatMap(t =>
+                Target.Sidereal.integratedSpectralDefinition
+                  .getOption(t)
+                  .orElse(Target.Sidereal.surfaceSpectralDefinition.getOption(t))
+              )
+              .map(_.shortName),
+          ColumnNames(SEDColumnId)
+        ).setCell(cell =>
+          cell.value
+            .filterNot(_ => cell.row.getCanExpand())
+            .orEmpty
+        ).sortable,
+        obsColumn(ConstraintsColumnId, r => (r.obs.id, r.obs.constraints.summaryString))
+          .setCell: cell =>
+            cell.value.map: (id, constraintsSummary) =>
+              <.a(
+                ^.href := constraintUrl(id),
+                ^.onClick ==> (_.preventDefaultCB *> goToConstraint(id)),
+                constraintsSummary
               )
-      .useReactTableWithStateStoreBy: (props, ctx, cols, rows) =>
-        import ctx.given
-
-        val obsIds2RowSelection: Iso[List[Observation.Id], RowSelection] =
-          Iso[List[Observation.Id], RowSelection](obsIds =>
-            RowSelection:
-              obsIds.map(obsId => RowId(obsId.toString) -> true).toMap
-          )(selection =>
-            selection.value
-              .filter(_._2)
-              .keys
-              .toList
-              .map(rowId => Observation.Id.parse(rowId.value))
-              .flattenOption
-          )
-
-        val rowSelection: View[RowSelection] = props.selectedObsIds.as(obsIds2RowSelection)
-
-        TableOptionsWithStateStore(
-          TableOptions(
-            cols,
-            rows,
-            enableExpanding = true,
-            getSubRows = (row, _) => row.subRows,
-            getRowId = (row, _, _) =>
-              RowId:
-                row.value.fold(
-                  o => o.obs.id.toString + o.targetWithId.id.toString,
-                  _.obs.id.toString
-                )
+          .sortableBy(_.map(_._2)),
+        // TODO: FindingChartColumnId
+        obsColumn(ConfigurationColumnId, _.obs.configurationSummary.orEmpty)
+          .setCell(_.value.orEmpty)
+          .sortable,
+        obsColumn(
+          DurationColumnId,
+          _.execution.map(_.programTimeEstimate)
+        ).setCell:
+          _.value.map:
+            _.orSpinner(_.map(HoursMinutesAbbreviation.format).orEmpty)
+        .sortableBy(_.sortableTimeSpan)
+        // TODO: PriorityColumnId
+        // TODO: ChargedTimeColumnId
+      )
+    end columns
+
+    private val component = ScalaFnComponent[Body]: props =>
+      for {
```

Comment:
I still can't get over how much cleaner this looks 😅

Reply:
Yeah. The old style is hard to look at now. 😄